docs/guide/web-simulation.md

```diff
@@ -452,30 +445,30 @@ Also the robot windows are not yet supported.
-  * URL to a X3D file (i.e. "file.x3d"): this format is used for showing a [web scene](web-scene.md) or a [web animation](web-animation.md)).
-* `webots.View.setTimeout(timeout)`: utility function to specify how long the simulation can run without user interaction. Setting a timeout value is useful to save resources on the server side and avoid having very long simulation running continuously on the machines and saturating the machines. The `timeout` value is expressed in seconds.
-* `webots.View.setWebotsDocUrl(url)`: utility function to specify the URL of the Webots documentation shown in the Help window.
+Similarly to [this section](web-streaming.md#how-to-embed-a-web-scene-in-your-website), to embed the simulation it is enough to instantiate a `webots-streaming` web component from the [WebotsStreaming.js] package.
+
+This is the API of the `webots-streaming` web component:
+* `connect(url, mode, broadcast, mobileDevice, callback, disconnectCallback) `: function instantiating the simulation web interface and taking as argument:
+  * `url`: Three different `url` formats are supported:
+      * URL to a WBT file (i.e. "ws://localhost:80/simple/worlds/simple.wbt"): this is the format required to start a web simulation. The `url` value specifies both the session server host and the desired simulation name.
+      * WebSocket URL (i.e. "ws://localhost:80"): this format is used for web broadcast streaming.
+      * URL to a X3D file (i.e. "file.x3d"): this format is used for showing a [web scene](web-scene.md) or a [web animation](web-animation.md)).
+  * `mode`: `x3d` or `mjpeg`.
+  * `broadcast`: boolean variable enabling or not the broadcast.
+  * `mobileDevice`: boolean variable specifying if the application is running on a mobile device.
+  * `callback`: function to be executed once the simulation is ready.
+  * `disconnectCallback`: function to be executed once the web scene is closed.
+* `disconnect()`: close the simulation web scene.
+* `hideToolbar()`: hide the toolbar. Must be called after connect.
+* `showToolbar()`: show the toolbar. Must be called after connect. The toolbar is displayed by default.
+* `showQuit(enable)`: specify is the quit button must be displayed on the toolbar. Must be called before connect. The quit button is displayed by default.
+* `showRevert(enable)`: specify is the revert button must be displayed on the toolbar. Must be called before connect. The quit button is hidden by default.
+* `sendMessage(message)`: send a message to the streaming server through the web socket. Examples of messages could be:
+    *`real-time:-1`: to play the simulation.
+    *`pause`: to pause the simulation.
+    *`robot:{"name":"supervisor","message":"reset"}`: to send a message to the controller of a robot named supervisor.
+
+An example of a file using this API is available [here](https://cyberbotics1.epfl.ch/open-roberta/setup_viewer.js) and is used to run [this sample](https://cyberbotics1.epfl.ch/open-roberta/).
```

Comment:
The example doesn't seem to work.
I I open it and click on connect i can just see the "Starting simulation" message but nothing happens.

Reply:
You are right, the server seemed to have a problem but it should be work now